.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Release
+about: Release checklist for maintainers
+title: 'Release X.Y.Z'
+labels: ''
+assignees: ''
+
+---
+
+[ ] Check out the `master` branch and ensure it contains all commits to be released
+
+[ ] Update CHANGELOG.md and version.sbt to the release version as a single commit to `master`. [Example](https://github.com/locationtech/geotrellis/commit/e8fbbe2668aa0e99b7a631e9dc7802bc270c2bda)
+
+[ ] Create a new tag at the previous commit with `git tag -a vX.Y.Z -m "vX.Y.Z"`
+
+[ ] Push the tag to `locationtech/geotrellis` with `git push --tags`
+
+[ ] Follow the instructions in the setup section of [publish/README.md](https://github.com/locationtech/geotrellis/blob/master/publish/README.md#setup) to ensure that your local environment is configured for publishing the release
+
+[ ] Set the `RELEASE_TAG` variable in `./publish/Makefile` to `vX.Y.Z`, substituting x/y/z for the current release version
+
+[ ] **Before Continuing:** Ensure you have pushed the tag you created in step 3 before continuing. Otherwise the next step will fail.
```

Comment:
I guess steps should be numbered?

Reply:
Looks like we haven't done this for other release templates and the markdown is readable without. I'd prefer to remain consistent with other Azavea projects unless there's a clear win we get for adding them.